tools/locale.mjs

```diff
@@ -61,7 +61,10 @@ const csvTranslationMap = (source) => {
     return {};
   }
 
-  const csvData = readFileSync(join(csvPath, importCSV)).toString();
+  const csvData = readFileSync(
+    // User can choose to read from an absolute path, or a relative path
+    importCSV.startsWith("/") ? importCSV : join(csvPath, importCSV),
```

Comment:
Looks like not work for Windows. How about the `isAbsolute` function in the `path` module?